src/web/app/components/question-types/question-statistics/mcq-question-statistics.component.ts

```diff
@@ -62,7 +66,7 @@ export class McqQuestionStatisticsComponent extends McqQuestionStatisticsCalcula
       { header: 'Recipient Name', sortBy: SortBy.MCQ_RECIPIENT_NAME },
       ...Object.keys(this.weightPerOption).map((key: string) => {
         return {
-          header: `${key} [${this.weightPerOption[key]}]`,
+          header: `${key} [${this.changeSignificantDigits.transform(this.weightPerOption[key], '1.2-3')}]`,
```

Comment:
I'm not keen on the use of this string variable in-line, feels too much like a magic number. Could you maybe define a constant variable with a descriptive name? Preferably in the scope of this method.